fhir-search/src/main/java/com/ibm/fhir/search/parameters/QueryParameterValue.java

```diff
@@ -150,6 +151,22 @@ public void setHidden(boolean hidden) {
         this.hidden = hidden;
     }
 
+    /**
+     * Sets whether the value of an :of-type modifier.
+     * @return true or false
+     */
+    public boolean isOfTypeModifier() {
+        return ofTypeModifier;
+    }
+
+    /**
+     * Gets whether the value of an :of-type modifier.
+     * @param ofTypeModifier true if value of an :of-type modifier
+     */
+    public void setOfTypeModifier(boolean ofTypeModifier) {
+        this.ofTypeModifier = ofTypeModifier;
+    }
```

Comment:
i think you mixed up the javadoc on these

Reply:
Fixed.